graphql_compiler/compiler/filters.py

```diff
@@ -89,10 +89,11 @@ def _is_tag_argument(argument_name):
     return argument_name.startswith('%')
 
 
-def _represent_argument(context, argument, inferred_type):
+def _represent_argument(directive_location, context, argument, inferred_type):
     """Return a two-element tuple that represents the argument to the directive being processed.
 
     Args:
+        directive_location: Location where this directive is used.
```

Comment:
super nit: `this` in `this directive` throws me off a bit here. Contextually, the reader knows we're in the middle of processing a directive (in the caller function), but the directive is sort of irrelevant and not passed into this function, so referring to it as `this` seems a bit strange. Maybe use `the` instead?